Justfile

```diff
@@ -3,18 +3,33 @@ import 'Justfile.dev'
 
 spec := "cypress/e2e/"
 trunk_port := `echo $((5170 + $RANDOM % 10))`
+indexer_port := "8081"
 
 export RUSTFLAGS := "--cfg=web_sys_unstable_apis"
 export CYPRESS_BASE_URL := 'http://localhost:' + trunk_port
 export VERSION := `git rev-parse --short=8 HEAD`
+export INDEXER_VERSION := `cd lib/mina-indexer && git rev-parse --short=8 HEAD`
```

Comment:
question(non-blocking): Does this need a slash or a dash between the 2 commands??

Reply:
It works as is. So i'd say no? But maybe that's a better style decision?